### PR TITLE
Update docs to remove deprecated references

### PR DESCRIPTION
**Pull request by oceanc80**
Update docs to remove deprecated references and replace with the correct references.

### CODE REVIEW

File: website/content/en/docs/building-operators/ansible/development-tips.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ sudo dnf install ansible
 ```
 
 In addition to Ansible, a user must install the
-[OpenShift Restclient Python][openshift-restclient-python] package:
+[Python Kubernetes Client][python-kubernetes-client] package:
 
 ```sh
 pip3 install openshift
```
Comment (everettraven) [inline website/content/en/docs/building-operators/ansible/development-tips.md:37]: Does this need to be a different command or removed?

File: website/content/en/docs/building-operators/ansible/testing-guide.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ To begin, you should have:
 - Docker installed and running
 - [Molecule](https://github.com/ansible/molecule) >= v3.0
 - [Ansible](https://github.com/ansible/ansible) >= v2.9
-- [The OpenShift Python client](https://github.com/openshift/openshift-restclient-python) >= v0.8
+- [The Python Kubernetes client](https://github.com/kubernetes-client/python)
```
Comment (everettraven) [inline website/content/en/docs/building-operators/ansible/testing-guide.md:16]: Is there a specific version that should be used or is "latest" okay?

**Review by everettraven — COMMENTED** [review]
Couple of questions:

File: website/content/en/docs/building-operators/ansible/development-tips.md (hunk 1 above)
Comment (asmacdo) [reply website/content/en/docs/building-operators/ansible/development-tips.md:37]: @fabianvf I did a lazy-quick search, do you remember what openshift client was needed for?

Comment (everettraven) [reply website/content/en/docs/building-operators/ansible/development-tips.md:37]: FWIW We are planning to remove it from the base container in a follow-up PR. We had some conversations with some folks working on Ansible that said all the functionality needed should be in the `kubernetes.core` collection now and that the openshift restclient is deprecated

Comment (oceanc80) [reply website/content/en/docs/building-operators/ansible/development-tips.md:37]: Oops, yeah, def missed that reference.

File: website/content/en/docs/building-operators/ansible/testing-guide.md (hunk 2 above)
Comment (oceanc80) [reply website/content/en/docs/building-operators/ansible/testing-guide.md:16]: I think this is okay for now and we can update it with the version that we use in your [PR](https://github.com/operator-framework/operator-sdk/pull/6483)

**Review by everettraven — APPROVED** [review]
/lgtm

**Pull request merged**